mixer/pkg/perf/load.go

```diff
@@ -129,7 +132,11 @@ func (l *Load) createRequestProtos(c Config) []interface{} {
 
 	// Shuffle requests if StableOrder is not explicitly requested.
 	if !l.StableOrder {
-		source := rand.NewSource(time.Now().UnixNano())
+		seed := l.RandomSeed
+		if seed == 0 {
+			seed = time.Now().UnixNano()
```

Comment:
I think we should what was the seed created, in case not specified, for use to reproduce and debug issues.